source/less/blocks/recall-form.less

```
@@ -0,0 +1,116 @@
+.recall-form__info {
+  padding: 0;
+  margin: 0;
+  border: none;
+  padding: 24px 20px 34px;
+}
+
+.recall-form__title {
+  font-family: "PT Sana", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 26px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #000000;
+  padding: 0;
+}
+
+.recall-form-question {
+  font-family: "PT Sans", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  text-transform: uppercase;
+  color: #333333;
+}
+
+.recall-form__list {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.recall-form__item:nth-child(2n) {
+  margin-bottom: 16px;
+}
+
+
+.recall-form-data-input {
```

```suggestion
.recall-form__data-input {
```
Это класс элемента, а не блока.